package.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
-    "jquery-datetimepicker": "^2.5.21"
+    "jquery-datetimepicker": "^2.5.21",
```

Comment:
Probably good to explicitly declare jQuery as a dependency:

```
warning " > multiple-select@1.5.2" has unmet peer dependency "jquery@1.9.1 - 3".
 ```

We do _have_ a version of jQuery that matches, but yarn is a little unsure to start out with.

Reply:
To declare this dependency you just straight up add it to the package.json right? then it knows

Reply:
it

Reply:
Yup.  Just `yarn add jquery` will get you 3.4.1 (the most recent version)

(Which is, weirdly, included in "1.9.1-3")